### PR TITLE
docs:  dotnet new in getting started with Toolkit

### PR DESCRIPTION
**Pull request by Darsh0307**
GitHub Issue (If applicable): closes https://github.com/unoplatform/uno.toolkit.ui/issues/972

## PR Type

What kind of change does this PR introduce?

## PR Checklist

Please check if your PR fulfills the following requirements:
- [ ] Tested code with current [supported SDKs](../README.md#supported)
- [ ] Tested the changes where applicable:
	- [ ] UWP
	- [ ] WinUI
	- [ ] iOS
	- [ ] Android
	- [ ] WASM
	- [ ] MacOS
- [ ] Updated the documentation as needed:
	-  [x] [General Doc Update](https://github.com/unoplatform/uno.toolkit.ui/tree/main/doc)
	- [ ] [Controls Doc Update](https://github.com/unoplatform/uno.toolkit.ui/tree/main/doc/controls)
	- [ ] [Extensions Doc Update](https://github.com/unoplatform/uno.toolkit.ui/tree/main/doc/helpers)
	- [ ] [controls-styles.md](https://github.com/unoplatform/uno.toolkit.ui/blob/main/doc/controls-styles.md)
	- [ ] [lightweight-styling.md (LightWeight Styling Resource Keys)](https://github.com/unoplatform/uno.toolkit.ui/blob/main/doc/lightweight-styling.md)
- [ ] [Runtime Tests and/or UI Tests](https://platform.uno/docs/articles/contributing/guidelines/creating-tests.html) for the changes have been added (for bug fixes / features) (if applicable)
- [ ] Contains **NO** breaking changes
- [ ] Associated with an issue (GitHub or internal)
- [ ] Commits must be following the [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/#summary) specification.

## Other information

Internal Issue (If applicable):

### CODE REVIEW

File: doc/getting-started.md [hunk 1]
```diff
@@ -15,7 +15,13 @@ This article is a guide for installing the base Uno.Toolkit library, additional
 - [Getting Started with Material for Toolkit](./material-getting-started.md)
 - [Getting Started with Cupertino for Toolkit](./cupertino-getting-started.md)
 
-### Installation
+## Installation
+
+I created the  project using
```
Comment (kazo0) [inline doc/getting-started.md:20]:
```suggestion
The Uno Toolkit library can be included in your application by using the [Visual Studio Wizard](xref:Uno.GettingStarted.CreateAnApp.VS2022) and/or the [`dotnet new` templates](xref:Uno.GetStarted.dotnet-new)
```

File: doc/getting-started.md [hunk 2]
```diff
@@ -35,6 +41,16 @@ This article is a guide for installing the base Uno.Toolkit library, additional
         </ResourceDictionary.MergedDictionaries>
     </ResourceDictionary>
     ```
+### [**dotnet new**](#tab/dotnet-new)
+
+1. When creating a new project using [dotnet new template](https://platform.uno/docs/articles/get-started-dotnet-new.html?tabs=windows#uno-platform-application) ensure that you include `-toolkit` in dotnet new  command . For example  `dotnet new unoapp -preset blank -o MyApp -toolkit`. Including the `-toolkit` option in the command ensures proper configuration of the Uno Toolkit for the newly created application.
```
Comment (kazo0) [inline doc/getting-started.md:46]:
```suggestion
1. When creating a new project using the `dotnet new` templates, make sure to include the [`-toolkit` option](xref:Uno.GettingStarted.UsingWizard.Features#toolkit) in the `dotnet new unoapp` command.

`dotnet new unoapp -preset blank -o MyApp -toolkit` 
```

File: doc/getting-started.md [hunk 3]
```diff
@@ -35,6 +41,16 @@ This article is a guide for installing the base Uno.Toolkit library, additional
...
+
+2. To incorporate the Uno Toolkit into an existing project, navigate to the **App Code Library** in the Terminal. Once in the command line interface, execute the following command:`dotnet add package Uno.Toolkit.WinUI --version [latest version number]`. Ensure to replace `[latest version number] `with the actual latest version number, which can be obtained from [Uno.Toolkit.UI](https://www.nuget.org/packages/Uno.Toolkit.UI). This command adds the Uno Toolkit package to your project.
```
Comment (kazo0) [inline doc/getting-started.md:48]:
```suggestion
2. To incorporate the Uno Toolkit into an existing application, you must add the [Uno.Toolkit.WinUI NuGet package](https://www.nuget.org/packages/Uno.Toolkit.WinUI) to the **App Code Library** project as well as the platform heads. An easy way to do this is through the command line. Run the following command within the directory of each `.csproj`:

`dotnet add package Uno.Toolkit.WinUI --version [latest version number]`. 

Make sure to replace `[latest version number] `with the actual latest NuGet version number.
```

File: doc/getting-started.md [hunk 4]
```diff
@@ -74,6 +90,8 @@ If your application is based on the older solution template that includes a shar
     </Application>
     ```
 
+    
+
```
Comment (kazo0) [inline doc/getting-started.md:94]: Remove whitespace

**Comment by Darsh0307** [comment]
@kazo0  @agneszitte  

Please have a look at this changes 

Here is the preview version https://github.com/unoplatform/uno.toolkit.ui/blob/main/doc/getting-started.md

**Comment by Darsh0307** [comment]
@kazo0  :- Here is the updated one :)

File: doc/getting-started.md [hunk 5]
```diff
@@ -35,6 +41,20 @@ This article is a guide for installing the base Uno.Toolkit library, additional
         </ResourceDictionary.MergedDictionaries>
     </ResourceDictionary>
     ```
+### [**dotnet new**](#tab/dotnet-new)
```
Comment (Xiaoy312) [inline doc/getting-started.md:44]: you shouldn't use numbered items here
it is not step1 followed by step2, but rather option1 vs option2

Comment (Xiaoy312) [reply doc/getting-started.md:44]: and, for "2." you need to included the part that adds the \<ToolkitResources> (step.4 from the Wizard section)

**Comment by Darsh0307** [comment]
https://github.com/unoplatform/uno.toolkit.ui/blob/dev/dp/GetStart/doc/getting-started.md

File: doc/getting-started.md [hunk 6]
```diff
@@ -35,6 +41,36 @@ This article is a guide for installing the base Uno.Toolkit library, additional
...
+
+            <!-- Load Uno.Toolkit.UI resources -->
+            <ToolkitResources xmlns="using:Uno.Toolkit.UI" />
+
+            <!-- Load custom application resources -->
+            <!-- ... -->
+
+        </ResourceDictionary.MergedDictionaries>
+    </ResourceDictionary>
+    ```
```
Comment (Xiaoy312) [inline doc/getting-started.md:70]: image

pick a side:
- either make everything on indentation level 1 and on code level 2
- or keep `option x:` on lv1, each step on lv2, and code on lv3

File: doc/getting-started.md [hunk 7]
```diff
@@ -35,6 +41,36 @@ This article is a guide for installing the base Uno.Toolkit library, additional
...
+
+***
+
 
 #### Previous Installation Method
```
Comment (Xiaoy312) [inline doc/getting-started.md:75]: i would argue that "previous installation method" on the same heading level as "installation"
if you have chosen heading 2 `##` for installation, then do the same for other.
```suggestion
## Previous Installation Method
```

File: doc/getting-started.md (hunk 6 above)
Comment (Xiaoy312) [reply doc/getting-started.md:70]: also, notice how the xml code snippet is not picking up the syntax, showing the "\`\`\`xml" part?
you either want to indent by a level or use \`\`\`, not both

**Comment by agneszitte** [comment]
@kazo0 based on the latest changes you made for the documentation, I think we can close this PR right?

**Comment by agneszitte** [comment]
Closing this PR as latest changes were made with https://github.com/unoplatform/uno.toolkit.ui/pull/1110

**Pull request closed without merging**